products/chromeos.md

```diff
@@ -0,0 +1,150 @@
+    eol: 2023-07-12
+
+-   releaseCycle: "113"
+    releaseDate: 2023-04-26
+    eol: 2023-05-24
+
+-   releaseCycle: "112"
+    releaseDate: 2023-03-29
+    eol: 2023-04-26
+
+-   releaseCycle: "111"
+    releaseDate: 2023-03-01
+    eol: 2023-03-29
+
+-   releaseCycle: "110"
+    releaseDate: 2023-02-01
+    eol: 2023-03-01
+
+-   releaseCycle: "109"
+    releaseDate: 2023-01-10
+    eol: 2023-02-01
+
+-   releaseCycle: "108"
+    releaseDate: 2022-11-29
+    lts: true
+    eol: 2023-03-01
+
+-   releaseCycle: "107"
+    releaseDate: 2022-10-25
+    eol: 2022-11-29
+
+-   releaseCycle: "106"
+    releaseDate: 2022-09-27
+    eol: 2022-10-25
+
+-   releaseCycle: "105"
+    releaseDate: 2022-08-30
+    eol: 2022-09-27
+
+-   releaseCycle: "104"
+    releaseDate: 2022-08-02
+    eol: 2022-08-30
+
+-   releaseCycle: "103"
+    releaseDate: 2022-06-21
+    eol: 2022-08-02
+
+-   releaseCycle: "102"
+    releaseDate: 2022-05-26
+    lts: true
+    eol: 2022-11-29 
+
+-   releaseCycle: "101"
+    releaseDate: 2022-04-26
+    eol: 2022-05-26
+
+-   releaseCycle: "100"
+    releaseDate: 2022-03-29
+    eol: 2022-04-26
+
+-   releaseCycle: "99"
+    releaseDate: 2022-03-01
+    eol: 2022-03-29
+
+-   releaseCycle: "98"
+    releaseDate: 2022-02-02
+    eol: 2022-03-01
+
+-   releaseCycle: "97"
+    releaseDate: 2022-01-04
+    eol: 2022-02-02
+
+-   releaseCycle: "96"
+    releaseDate: 2021-12-06
+    eol: 2022-01-04
+
+-   releaseCycle: "94"
+    releaseDate: 2021-09-21
+    eol: 2021-12-06
+
+-   releaseCycle: "93"
+    releaseDate: 2021-08-31
+    eol: 2021-09-21
+
+-   releaseCycle: "92"
+    releaseDate: 2021-07-20
+    eol: 2021-08-31
+
+-   releaseCycle: "91"
+    releaseDate: 2021-05-21
+    eol: 2021-07-20
+
+-   releaseCycle: "90"
+    releaseDate: 2021-04-14
+    eol: 2021-05-21
+
+-   releaseCycle: "89"
+    releaseDate: 2021-03-09
+    eol: 2021-04-14
+
+-   releaseCycle: "88"
+    releaseDate: 2021-01-19
+    eol: 2021-03-09
+
+-   releaseCycle: "87"
+    releaseDate: 2020-11-17
+    eol: 2021-01-19
+
+-   releaseCycle: "86"
+    releaseDate: 2020-10-06
+    eol: 2020-11-17
+
+-   releaseCycle: "85"
+    releaseDate: 2020-08-25
+    eol: 2020-10-06
+
+---
+> [ChromeOS](https://www.google.com/chromebook/chrome-os/) is a Linux-based operating system developed and designed by Google. It is derived from the open-source Chromium OS and uses the Google Chrome web browser as its principal user interface.
+
+According to [Auto Update policy](https://support.google.com/chrome/a/answer/6220366?hl=en), Chrome devices (e.g. Chromebook, Chromebox, Chromebase, Chromebit) receive automatic updates that enhance both the device and its software. Automatic updates provide the latest features, keep the device secure, and are applied across the operating system, browser and hardware. These updates depend on many device specific non-Google hardware and software providers that work with Google to provide the highest level of security and stability support.
```

Comment:
This description is about devices, not ChromeOS per se.

What's written on the previous page you used looks more focus on ChromeOS: https://chromeos.dev/en/education/chromeos-lts and provide a little bit more insights:

- stable : 1 release every 4 weeks
- LTS : 1 per 6 months

It does not say how much time stable and LTS releases are supported though. Maybe a stable is supported until the next, and similarly an LTS is supported until the next LTS ?